### ISSUE TITLE
ELI5 Installation

### ISSUE THREAD

**Issue by cruzjrp** [thelinmichael/spotify-web-api-java#183]
As a beginner to APIs and API wrappers in general, reading the Installation section of the readme is quite confusing and I'm not sure what the process is for installation because I don't know any of the software in that section. Is there any way I could get an Installation for Dummies? I'd really love to use this wrapper for a project.

**Comment by dargmuesli** [comment]
Hey there! It's great to hear that you want to start getting into APIs.

> I don't know any of the software in that section

Given that, I think this repository is the wrong place to start though. Do you have experience programming in Java?

**Comment by cruzjrp** [comment]
Yeah I graduated in CS using Java for most of my courses. We've just never touched APIs in the courses though for some reason.

**Comment by dargmuesli** [comment]
Ok, then the only missing piece seems to be Gradle / Maven. But I'm afraid that the best way to learn about those is for you to toss 'em into Google. Currently I have no better material at hand.

**Comment by cruzjrp** [comment]
Okay, I installed Gradle. What do I do at this point to install the wrapper? Not sure what this line is supposed to mean:  

> compile 'se.michaelthelin.spotify:spotify-web-api-java:4.2.0'

**Comment by dargmuesli** [comment]
It's a dependency. Compare it to this file for example: https://github.com/dargmuesli/rbsg/blob/master/build.gradle
I'm sorry, but I think I'll close this as I'm pretty short on time currently. I'm sure you'll find other helpful content on the web. Good luck!
